eth2deposit/intl/zh-CN/cli/generate_keys.json

```
@@ -0,0 +1,35 @@
+{
+    "validate_password": {
+        "msg_password_prompt": "输入密码以保护您的验证者密钥库",
+        "msg_password_confirm": "重复以确认",
+        "err_password_mismatch": "错误：输入的两个值不匹配。请再次输入。"
+    },
+    "generate_keys_arguments_decorator": {
+        "num_validators": {
+            "argument": "--num_validators",
+            "help": "您想要生成的验证者密钥数量（您以后随时可以生成更多密钥）",
+            "prompt": "请选择您想要运行多少验证者"
+        },
+        "folder": {
+            "argument": "--folder",
+            "help": "您想要生成的验证者密钥数量（您以后随时可以生成更多密钥）"
+        },
+        "chain": {
+            "argument": "--chain",
+            "help": "您正在对准的 eth2 版本。如果您要存入以太币，请使用“主网”",
```

It was wrong since "mainnet" should not be translated here:
```suggestion
            "help": "您正在对准的 eth2 版本。如果您要存入以太币，请使用 \"mainnet\"",
```